support-frontend/assets/helpers/forms/paymentMethods.ts

```diff
@@ -17,12 +17,17 @@ export type PaymentMethodMap<T> = {
 	None: T;
 	AmazonPay: T;
 };
-// This lets us create a union type from the object keys,
-// avoiding the need to specify them separately and keep them in sync!
-// https://flow.org/en/docs/types/utilities/#toc-keys
-// We need to supply the type parameter, but we're only using the keys
-// so it's irrelevant - so we supply null
-export type PaymentMethod = $Keys<PaymentMethodMap<null>>;
+
+export type PaymentMethod =
```

Comment:
could this be simpler now?
`export type PaymentMethod = 'Stripe' | 'PayPal' | ...`

Reply:
actually, I'm guessing there are loads of places that import those consts, which would make it a very big PR...

Reply:
Yes that's the reason I didn't get rid of `PaymentMethodMap` as well even though I don't think we need it now - it was basically working round some limitations of Flow. 
Let's save those for another PR!